"""
Custom exceptions module.
"""


class _OutOfMoneyError(Exception):
    pass
